client/functions.lua

```diff
@@ -362,6 +362,19 @@ function QBCore.Functions.GetBoneDistance(entity, Type, Bone)
     return #(boneCoords - playerCoords)
 end
 
+function QBCore.Functions.attachProp(ped, model, boneId, x, y, z, xR, yR, zR, Vertex)
```

Comment:
This doesn't match the naming conventions and it upsets me, please change to AttachProp